src/mongocrypt-kms-ctx.c

```diff
@@ -1376,6 +1451,9 @@ _mongocrypt_kms_ctx_init_gcp_decrypt (
    _mongocrypt_key_doc_t *key,
    _mongocrypt_log_t *log)
 {
+   BSON_ASSERT_PARAM (kms);
+   BSON_ASSERT_PARAM (key);
```

Comment:
```suggestion
   BSON_ASSERT_PARAM (kms);
   BSON_ASSERT_PARAM (kms_providers);
   BSON_ASSERT_PARAM (access_token);
   BSON_ASSERT_PARAM (key);
```
Assert `kms_providers` and `access_token` as not-null.